src/components/AddNewComment.jsx

```diff
@@ -0,0 +1,61 @@
+import { useContext, useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { addCommentToArticleByID } from "../utils/api";
+import { UserContext } from "../contexts/UserContext";
+import { Link } from "react-router-dom";
+
+function AddNewComment({ comments, setComments }) {
+  const { user } = useContext(UserContext);
```

Comment:
The `user` is a great use case for context, nice one
I like that you've logged someone in as a default - obviously not what you'd do in a live app but great for ease of use of your portfolio piece 👍